spec/acceptance/mod_dav_svn_spec.rb

```diff
@@ -0,0 +1,18 @@
+# frozen_string_literal: true
+
+require 'spec_helper_acceptance'
+
+describe 'apache::mod::dav_svn class' do
+  context 'dav_svn module with authz_svn_enabled' do
+    pp = <<-MANIFEST
+      class { 'apache': }
+      class { 'apache::mod::dav_svn':
+        authz_svn_enabled => true,
+      }
+    MANIFEST
+
+    it 'succeeds in installing the dav_svn module' do
+      apply_manifest(pp, catch_failures: true)
+    end
```

Comment:
Could you add an idempotency check to this?

Reply:
I’m not sure what you mean by this. Do you want to run the apply_manifest twice? To check if it fails/passes 2 times in a row?

Reply:
Copied from another module:
```ruby
  it 'applies with no errors' do
    apply_manifest(pp, catch_failures: true)
  end

  it 'applies a second time without changes' do
    apply_manifest(pp, catch_changes: true)
  end
```